business_logic/db/experimentsCollection.js

```diff
@@ -26,23 +26,64 @@ var experimentsCollection_global = null
 
 async function loadExperimentsCollection(database) {
   try {
-    if (!experimentsCollection_global){
+    if (!experimentsCollection_global) {
       experimentsCollection_global = await database.collection('Experiments');
     }
   }
   catch {
-      return null;
+    return null;
   }
 }
 
 
-  //delete the last experiment and insert the new one
-  function insertExperiment (experiment){
-    collection.remove({});
-    collection.insert(experiment);
+//delete the last experiment and insert the new one
+function insertExperiment(experiment) {
+  collection.remove({});
+  collection.insert(experiment);
+}
+function getExperiment(expId) {
+  return experimentsCollection.find({ exp_id: expId });
+}
+//remove after hackhton
+function getExperiments() {
+  return collection.find({});
+}
+
+function getExperimentByCode(expCode) {
+  return collection.find({ exp_code: expCode });
+}
+
+//what i need to insert? username
+//to specific exp or just for hackthon the only exp - specific
+function insertParticipant(expId, participant) {
+  var parsedData = JSON.parse(participant);
+  var username;
+  var groupId;
+  for (item in parsedData) {
+    if (item == "participant_twitter_username") {
+      username = parsedData[item];
+    }
+    if (item == "group_id") {
+      groupId = parsedData[item];
+    }
+
   }
```

Comment:
Maybe you simply meant: 
`username = participant.participant_twitter_username
groupId  = participant.group_id`

Reply:
fixed